docs/Topics/mqtt.md

```diff
@@ -0,0 +1,47 @@
+--8<-- "References/abbreviations.md"
+
+# Message Queuing Telemetry Transport (MQTT) Eventing Support
+
+Open AMT Cloud Toolkit supports Eventing, customizable event monitoring, using Message Queuing Telemetry Transport (MQTT), an IoT publish-and-subscribe network protocol. With MQTT Eventing administrators subscribe and customize network event monitoring. This eliminates the need to query or poll MPS to determine network events, such as a device's activation or deactivation. Administrators can subscribe to events and respond proactively. 
+
+!!! important
+    Currently, the implementation publishes all REST API call events to the MPS. 
+
+![../assets/images/MQTT.png](../assets/images/MQTT.png)
+**Figure 1: MQTT Eventing Examples**
+
+An Open AMT Cloud Toolkit sends JSON events to a Mosquitto* broker deployed as a Docker container. Administrators subscribe to the broker. In Figure 1 above, proactive notifications are published on the MQTT bus. 
+
+## What You'll Need
+
+**Obtain MQTT Explorer: **
```

Comment:
Is this for the DXA or more generalized documentation?  If it is more generic, then we may want to say something along the lines of: "pick the MQTT tool of your choice.  The example we use is MQTT Explorer."

Reply:
I thought it could work for either, so that is a worthwhile change.